app/views/shared/tags/_curated_links_preview.html.erb

```diff
@@ -17,7 +19,13 @@
       <% unless list_item.tagged? %>
         <p class='label label-warning'>Tag was removed</p>
       <% end %>
-      <%= raw list_item.display_title || "<em>Unknown title (#{list_item.base_path})</em>" %>
+      <% if list_item.base_path %>
+        <a href=<%= Plek.new.find('content-tagger') + '/taggings/lookup' + list_item.base_path %>>
+          <%= raw list_item.display_title || "<em>Unknown title (#{list_item.base_path})</em>" %>
```

Comment:
Perhaps this should be an explicit if condition?

```
<% if ... %>
<em>Unknown...</em>
<% else %>
list_item.display_title
<% end %>
```

I'm not too familiar with the template language but it looks like this could have XSS problems if the title contained special characters.